### PR TITLE
DEV2-7583: save/load storage state to detect commits for review

### PR DESCRIPTION
**Pull request by wyruweso**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -29,13 +29,25 @@ inputs:
 runs:
   using: 'composite'
   steps:
+
+    - name: Getting previous run state
```
Comment (igor-klymenok) [inline action.yml:33]: What will happen if we miss previous run state?

File: action.yml [hunk 2]
```diff
@@ -29,13 +29,25 @@ inputs:
...
+      with:
+        name: storage
+        path: /etc/tabnine_cli/storage.json
     - name: Pull docker image
       run: docker pull ghcr.io/codota/tabnine-cli:latest
       shell: bash
 
     - name: Run analyse
-      run: docker run -e GITHUB_TOKEN=${{ inputs.github_token }} -e GITHUB_ACTION=__self --rm -v ./tabnine:/etc/tabnine_cli -v ${{inputs.path_to_repo}}:/opt/repository ghcr.io/codota/tabnine-cli:latest review diff -b ${{ inputs.base_ref }} -bf ${{ inputs.ref }} --token ${{inputs.PAT}} --workspace ${{ inputs.repository_owner }} --repo_slug ${{ inputs.repository_name }} --pr_id ${{ inputs.pull_request_id }} /opt/repository
+      run: docker run -e --rm -v ./tabnine:/etc/tabnine_cli -v ${{inputs.path_to_repo}}:/opt/repository ghcr.io/codota/tabnine-cli:latest review diff -b ${{ inputs.base_ref }} -bf ${{ inputs.ref }} --token ${{inputs.PAT}} --workspace ${{ inputs.repository_owner }} --repo_slug ${{ inputs.repository_name }} --pr_id ${{ inputs.pull_request_id }} /opt/repository
```
Comment (igor-klymenok) [inline action.yml:43]: The "-e" argument is redundant here

```shell
docker run -e --rm
```

**Review by igor-klymenok — CHANGES_REQUESTED** [review]
I suggest to remove "-e" flag from the docker command run

File: action.yml (hunk 1 above)
Comment (wyruweso) [reply action.yml:33]: good point, I forgot to add continue-on-error. Fixed

File: action.yml (hunk 2 above)
Comment (wyruweso) [reply action.yml]: fixed

**Pull request merged**